ui/packages/catalog-ui-search/src/main/webapp/react-component/metacard-interactions/expand-interaction.tsx

```diff
@@ -48,8 +48,10 @@ const handleExpand = (props: Props) => {
 
   id = encodeURIComponent(id)
 
+  const federation = props.model.first().get('src') === 'Local Content Stores' ? 'local' : 'enterprise'
```

Comment:
I think this is a specific source name in a downstream project

Reply:
You can get the harvest source from one of our ui utils and check for it, like we did here https://github.com/connexta/gsr-yorktown/blob/master/ui/packages/catalog-ui-search-gsr/src/main/webapp/utils/add-to-cart-util.tsx#L31

Reply:
_the_ harvested source? cant there be more than one?